pennylane/templates/subroutines/uccsd.py

```diff
@@ -87,6 +87,7 @@ class UCCSD(Operation):
             and unoccupied orbitals in the intervals ``[s, r]`` and ``[q, p]``, respectively.
         init_state (array[int]): Length ``len(wires)`` occupation-number vector representing the
             HF state. ``init_state`` is used to initialize the wires.
+        reps (int): Number of times UCCSD unitary is repeated.
```

Comment:
We have used `n_repeats` in a few other templates. We could reuse that name here as well.